### ISSUE TITLE
Glitch in validation alert message for email acceptance

### ISSUE THREAD

**Issue by shahkv95** [legesher/legesher-dot-io#38]
The alert message which asks for an proper format for an email as the required 
has different alignments in both different locations. It must be similar to make it look tidy.

I don't know why is '0' printed in the validation message.

> '0 - An email address must contain a single @'

@madiedgar Please check this issue and accept and label it if you think it needs  to be solved.

Thank you.

**Comment by madiedgar** [comment]
Hmm, thanks for finding this! Definitely would love to get to the bottom of it, do you want to take it on?

**Comment by shahkv95** [comment]
@madiedgar Yeah I would love to work on it.
Can I work on it?

**Comment by shahkv95** [comment]
@madiedgar Could u please assign me for this issue?

**Comment by madiedgar** [comment]
Yes! That would be amazing! I'll assign you to it! 😄

**Comment by shahkv95** [comment]
Heyy @madiedgar , I need your help.
Since last couple of minutes I am trying to find whether where is this validation messages being coded..!
Messages:
"0 - Please enter a value"
"0 - An email address must contain a single @"
"0 - The domain portion of the email address is invalid (the portion after the @: )"
"0 - The domain portion of the email address is invalid (the portion after the @: a)"

Could you please help me to find the location for this statements, because I believe that these are dynamic statements which will showcase the error messages depending upon the error.

I think these are log errors so may be we won't be able change the text as they are prebuilt in javascript.

Thanking in anticipation..

Regards 
Kush

**Comment by madiedgar** [comment]
Maybe to give a little more insight, the email is done through mailchimp so looking at that api might be helpful. Most likely those errors are from their script that's included on the page and [`ajax-mailchimp.js`](https://github.com/legesher/legesher-dot-io/blob/master/src/js/ajax-mailchimp.js). Hopefully this helps!

**Comment by shahkv95** [comment]
Ma'am after checking all the codes even the one you mentioned, I texted you for the same.

@madiedgar I again made more efforts. I have checked all the **XHR and Fetch domain of Network** and also skimmed the **JS script networking domain**. I have come to a conclusion that that the sentences aren't hard coded. They are a part of the open source available scripts - JS and jQuery. 

**Output of XHR and Fetch Network domain Check** is as follows:

0 / 5 requests
0 B / 2.0 KB transferred
0 B / 19.3 KB resources
jQuery34109483622224247681_1572364542179({result: "error", msg: "0 - Please enter a value"})
msg: "0 - Please enter a value"
result: "error"

and following were the **outputs of the JS script of the network domain**:

jQuery34109483622224247681_1572364542181({result: "error", msg: "0 - Please enter a value"})
msg: "0 - Please enter a value"
result: "error"

jQuery34109483622224247681_1572364542181({result: "error", msg: "0 - An email address must contain a single @"})
msg: "0 - An email address must contain a single @"
result: "error"

jQuery34109483622224247681_1572364542179({result: "error",…})
msg: "0 - The domain portion of the email address is invalid (the portion after the @: )"
result: "error"

jQuery34109483622224247681_1572364542179({result: "error",…})
msg: "0 - The domain portion of the email address is invalid (the portion after the @: sdsd)"
result: "error"

**I have checked all the initiators and dependencies of a request.**

So basically it means that the is not a part of the hard coded scripts. To resolve this matter, the only possible way is to overwrite the scripts according in jQuery as well as JS if style is edited in JS. In that much time we could rather add some more functionalities to the code.

Thank you so much @madiedgar ma'am.
